src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -739,11 +764,16 @@ def _tulsi_sources_aspect(target, ctx):
     datamodels = _collect_xcdatamodeld_files(rule_attr, "datamodels")
     datamodels.extend(_collect_xcdatamodeld_files(rule_attr, "data"))
 
+    copts = None if is_swift_library else expanded_copts,
+    swiftc_opts = expanded_copts if is_swift_library else None,
```

Comment:
I don't think you want the trailing commas here, and it seems like there's some kind of duplication going on here between this lines and the argument changes below?

Reply:
Thanks for the comment @jparise - good point. I think this was introduced in a rebase. Let me fix this real quick